app/controllers/api/v1/service_days_controller.rb

```diff
@@ -5,12 +5,32 @@ module V1
     # API for user service_days
     class ServiceDaysController < Api::V1::ApiController
       before_action :set_service_days, only: %i[index]
+      before_action :set_service_day, only: %i[update destroy]
 
       # GET /service_days
       def index
         render json: ServiceDayBlueprint.render(@service_days)
       end
 
+      # PUT /service_days
+      def update
+        if @service_day.update(service_day_params)
+          render json: @service_day, status: :updated
+        else
+          render json: @service_day.errors, status: :unprocessable_entity
+        end
+      end
+
+      # DELETE /service_days
+      def destroy
+        result = @service_day.destroy
+        if result
+          render json: result, status: :destroyed
```

Comment:
If I remember correctly, the better response for destruction is `no_content` - check out how we're doing `destroy` in the AttendancesController for guidance here (it's using a command, so you could do:

```
      def destroy
        if @service_day
          @service_day.destroy
          render status: :no_content
        else
          render status: :not_found
        end
      end
```